src/applications/vaos/components/calendar/CalendarCheckboxOption.jsx

```diff
@@ -1,29 +1,47 @@
 import React from 'react';
+import classNames from 'classnames';
 
 const CalendarCheckboxOption = ({
-  id,
-  fieldName,
-  value,
   checked,
-  onChange,
+  disabled,
+  fieldName,
+  id,
   label,
-}) => (
-  <div className="vaos-calendar__option">
-    <input
-      id={`checkbox-${id}`}
-      type="checkbox"
-      name={fieldName}
-      value={value}
-      checked={checked}
-      onChange={onChange}
-    />
-    <label
-      className="vads-u-margin--0 vads-u-font-weight--bold vads-u-color--primary"
-      htmlFor={`checkbox-${id}`}
-    >
-      {label}
-    </label>
-  </div>
-);
+  onChange,
+  value,
+}) => {
+  const divClasses = classNames('vaos-calendar__option', {
+    disabled,
+  });
+
+  const labelClasses = classNames(
+    'vads-u-margin--0',
+    'vads-u-font-weight--bold',
+    {
+      'vads-u-color--primary': !disabled,
+      'vads-u-color--gray-lighter': disabled,
+    },
+  );
```

Comment:
Use CSS styles to make the checkboxes appear disabled.  Had to do this instead of adding the `disabled` attribute because we still want to be able to detect click events on the disabled checkboxes.

Reply:
I saw that when I was reviewing this morning. It validates to have `disabled` on the `<div>` even though we're _not exactly_ disabling it. I've done some fiddling with the CSS to add more contrast to the "disabled" text and make sure the not-allowed cursor extends to the entire label not just the input hit space. Attaching a screen shot with notes below. /cc @rtwell 

---
![css-notes-calendar-widget](https://user-images.githubusercontent.com/934879/76103671-acade780-5f97-11ea-8c0b-071bc398ccb4.jpg)